doc/cephfs/fs-volumes.rst

```diff
@@ -187,6 +187,10 @@ List cephx auth IDs authorized to access fs subvolume::
 
     $ ceph fs subvolume authorized_list <vol_name> <sub_name> [--group_name=<group_name>]
 
+Evict subvolume clients based on auth IDs::
```

Comment:
Evict fs clients based on auth ID and subvolume mounted

Reply:
Done